public/data/javascript.json

```diff
@@ -27,6 +27,34 @@
         ],
         "tags": ["javascript", "array", "flatten", "utility"],
         "author": "dostonnabotov"
+      },
+      {
+        "title": "Duplicate Count",
+        "description": "Count the duplicate values of an array",
+        "code": [
+          " function countDuplicates(arr) {",
+          "   const counts = {};",
+          "   arr.forEach((item) => {",
+          "     counts[item] = (counts[item] || 0) + 1;",
+          "   });",
+          " const duplicates = {};",
```

Comment:
You could replace the `duplicates` part with a simple filter over the object
```ts
Object.fromEntries(
    counts.entries().filter(([key, value]) => value > 1)
)
```
It makes the code easier to understand at a glance.